gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/resources/users/UsersEndpoint.java

```diff
@@ -150,7 +163,7 @@ public void create(RoutingContext context) {
 
             // schemas field is REQUIRED and MUST contain valid values and MUST not contain duplicate values
             try {
-                checkSchemas(user.getSchemas(), EnterpriseUser.SCHEMAS);
+                checkSchemas(user.getSchemas(), restrictedSchemas);
```

Comment:
Here we are performing schema validation using the schemas extracted from the payload. 
I wondering if we should create a constant of Supported Schemas (with User, Entreprise and Custom) and use it here :thinking:

Reply:
FP